packages/terra-application/src/application-status-overlay/ApplicationStatusOverlayProvider.module.scss

```diff
@@ -23,4 +23,8 @@
     width: 100%;
     z-index: 1;
   }
+
+  .remove-scroll {
+    overflow: visible;
```

Comment:
https://caniuse.com/?search=overflow%3A%20visible 
overflow-visible is not supported on IE 10 and IE 11 have we tested this on the IE11 and IE10 browsers.

Reply:
Since this is an MPage-specific change and MPages 9.0 supports only edge. We do not have to worry about IE.;